### PR TITLE
Release v3.3.2

### LINKED ISSUE #15: thread 'main' panicked if ./epic-wallet listen -m keybase is called
2022-02-15 20:11:06.414 ERROR 
thread 'main' panicked at 'called `Option::unwrap()` on a `None` value': epic-wallet/controller/src/command.rs:150   0: backtrace::backtrace::trace
   1: backtrace::capture::Backtrace::new
   2: epic_util::logger::send_panic_to_log::{{closure}}
   3: std::panicking::rust_panic_with_hook
             at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c/library/std/src/panicking.rs:628:17
   4: std::panicking::begin_panic_handler::{{closure}}
             at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c/library/std/src/panicking.rs:519:13
   5: std::sys_common::backtrace::__rust_end_short_backtrace
             at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c/library/std/src/sys_common/backtrace.rs:139:18
   6: rust_begin_unwind
             at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c/library/std/src/panicking.rs:517:5
   7: core::panicking::panic_fmt
             at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c/library/core/src/panicking.rs:100:14
   8: core::panicking::panic
             at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c/library/core/src/panicking.rs:50:5
   9: epic_wallet_controller::command::listen
  10: epic_wallet::cmd::wallet_args::wallet_command
  11: epic_wallet::cmd::wallet::wallet_command
  12: epic_wallet::real_main
  13: epic_wallet::main
  14: std::sys_common::backtrace::__rust_begin_short_backtrace
  15: std::rt::lang_start::{{closure}}
  16: core::ops::function::impls::<impl core::ops::function::FnOnce<A> for &F>::call_once
             at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c/library/core/src/ops/function.rs:259:13
      std::panicking::try::do_call
             at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c/library/std/src/panicking.rs:403:40
      std::panicking::try
             at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c/library/std/src/panicking.rs:367:19
      std::panic::catch_unwind
             at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c/library/std/src/panic.rs:133:14
      std::rt::lang_start_internal::{{closure}}
             at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c/library/std/src/rt.rs:128:48
      std::panicking::try::do_call
             at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c/library/std/src/panicking.rs:403:40
      std::panicking::try
             at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c/library/std/src/panicking.rs:367:19
      std::panic::catch_unwind
             at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c/library/std/src/panic.rs:133:14
      std::rt::lang_start_internal
             at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c/library/std/src/rt.rs:128:20
  17: _main

Thread 'main' panicked with message:
"called `Option::unwrap()` on a `None` value"

### PR DESCRIPTION
**Pull request by jhelison**
# Contents of the Release

- Fix #15: unwrap error on empty password for keybase listener #28
- LMDB to SQLite interface #34
- Emoji #32
- Fix/issue 20 #31
- Feat/outputs history #30
- LMDB to SQLite interface #34
- Refactor error handling on SQLite implementation #35
- Edit pre-commit and add PR template #36
- Add automation to build the release binaries and tar files #37
- Update main readme for the project #40
- Fixing simple warnings #41
- Add better error handling for not found #43
- Refactor error handling on SQLite implementation #35

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,11 +1,11 @@
 [package]
 name = "epic_wallet"
-version = "3.0.0"
+version = "3.3.2"
 authors = ["Epic Developers <mimblewimble@lists.launchpad.net>"]
```
Comment (Jualns) [inline Cargo.toml:4]: Change this author to Grin and Epic Developers

File: Cargo.toml [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
 description = "Simple, private and scalable cryptocurrency implementation based on the MimbleWimble chain format."
 license = "Apache-2.0"
 repository = "https://github.com/mimblewimble/epic-wallet"
```
Comment (Jualns) [inline Cargo.toml:7]: Change this repository to "https://github.com/EpicCash/epic-wallet"

File: api/Cargo.toml [hunk 3]
```diff
@@ -1,11 +1,11 @@
 [package]
 name = "epic_wallet_api"
-version = "3.0.0"
+version = "3.3.2"
 authors = ["Epic Developers <mimblewimble@lists.launchpad.net>"]
```
Comment (Jualns) [inline api/Cargo.toml:4]: Change this author to Grin and Epic Developers

File: api/Cargo.toml [hunk 4]
```diff
@@ -1,11 +1,11 @@
...
 description = "Epic Wallet API"
 license = "Apache-2.0"
 repository = "https://github.com/mimblewimble/epic-wallet"
```
Comment (Jualns) [inline api/Cargo.toml:7]: Change this repository to "https://github.com/EpicCash/epic-wallet"

File: config/Cargo.toml [hunk 5]
```diff
@@ -1,11 +1,11 @@
 [package]
 name = "epic_wallet_config"
-version = "3.0.0"
+version = "3.3.2"
 authors = ["Epic Developers <mimblewimble@lists.launchpad.net>"]
```
Comment (Jualns) [inline config/Cargo.toml:4]: Change this author to Grin and Epic Developers

File: config/Cargo.toml [hunk 6]
```diff
@@ -1,11 +1,11 @@
...
 description = "Configuration for epic wallet , a simple, private and scalable cryptocurrency implementation based on the MimbleWimble chain format."
```
Comment (Jualns) [inline config/Cargo.toml:5]: remove the space before ,

File: config/Cargo.toml [hunk 7]
```diff
@@ -1,11 +1,11 @@
...
 license = "Apache-2.0"
 repository = "https://github.com/mimblewimble/epic-wallet"
```
Comment (Jualns) [inline config/Cargo.toml:7]: Change this repository to "https://github.com/EpicCash/epic-wallet"

File: controller/Cargo.toml [hunk 8]
```diff
@@ -1,13 +1,12 @@
 [package]
 name = "epic_wallet_controller"
-version = "3.0.0"
+version = "3.3.2"
 authors = ["Epic Developers <mimblewimble@lists.launchpad.net>"]
 description = "Controllers for epic wallet instantiation"
 license = "Apache-2.0"
 repository = "https://github.com/mimblewimble/epic-wallet"
```
Comment (Jualns) [inline controller/Cargo.toml:7]: Change this repository to "https://github.com/EpicCash/epic-wallet"

File: controller/Cargo.toml [hunk 9]
```diff
@@ -1,13 +1,12 @@
 [package]
 name = "epic_wallet_controller"
-version = "3.0.0"
+version = "3.3.2"
 authors = ["Epic Developers <mimblewimble@lists.launchpad.net>"]
```
Comment (Jualns) [inline controller/Cargo.toml:4]: Change this author to Grin and Epic Developers

File: util/Cargo.toml [hunk 10]
```diff
@@ -1,11 +1,11 @@
 [package]
 name = "epic_wallet_util"
-version = "3.0.0"
+version = "3.3.2"
 authors = ["Epic Developers <mimblewimble@lists.launchpad.net>"]
```
Comment (Jualns) [inline util/Cargo.toml:4]: Change this author to Grin and Epic Developers

File: impls/Cargo.toml [hunk 11]
```diff
@@ -1,11 +1,11 @@
 [package]
 name = "epic_wallet_impls"
-version = "3.0.0"
+version = "3.3.2"
 authors = ["Epic Developers <mimblewimble@lists.launchpad.net>"]
 description = "Concrete types derived from libwallet traits"
 license = "Apache-2.0"
 repository = "https://github.com/mimblewimble/epic-wallet"
```
Comment (Jualns) [inline impls/Cargo.toml:7]: Change this repository to "https://github.com/EpicCash/epic-wallet"

File: libwallet/Cargo.toml [hunk 12]
```diff
@@ -1,13 +1,12 @@
 [package]
 name = "epic_wallet_libwallet"
-version = "3.0.0"
+version = "3.3.2"
 authors = ["Epic Developers <mimblewimble@lists.launchpad.net>"]
 description = "Simple, private and scalable cryptocurrency implementation based on the MimbleWimble chain format."
 license = "Apache-2.0"
 repository = "https://github.com/mimblewimble/epic-wallet"
```
Comment (Jualns) [inline libwallet/Cargo.toml:7]: Change this repository to "https://github.com/EpicCash/epic-wallet"

File: util/Cargo.toml [hunk 13]
```diff
@@ -1,11 +1,11 @@
...
 description = "Util, for generic utilities and to re-export epic crates"
 license = "Apache-2.0"
 repository = "https://github.com/mimblewimble/epic-wallet"
```
Comment (Jualns) [inline util/Cargo.toml:7]: Change this repository to "https://github.com/EpicCash/epic-wallet"

File: libwallet/Cargo.toml [hunk 14]
```diff
@@ -1,13 +1,12 @@
 [package]
 name = "epic_wallet_libwallet"
-version = "3.0.0"
+version = "3.3.2"
 authors = ["Epic Developers <mimblewimble@lists.launchpad.net>"]
```
Comment (Jualns) [inline libwallet/Cargo.toml:4]: Change this author to Grin and Epic Developers

**Review by Jualns — APPROVED** [review]
LGTM!

File: CHANGELOG.md [hunk 15]
```diff
@@ -0,0 +1,162 @@
+# epic-wallet 3.3.2
```
[the remark this answers is not in the document]
Comment (jhelison) [reply CHANGELOG.md:1]: Sure, that make sense, we have a copy of this fine on https://github.com/EpicCash/epic-wallet/blob/637920e8e61e6f91f8035ba29d6d6188ee1cb457/debian/changelog if we ever need it. (This one is important, it's used on the release notes for the deb file)

File: util/Cargo.toml (hunk 13 above)
Comment (jhelison) [reply util/Cargo.toml]: Done

File: libwallet/Cargo.toml (hunk 14 above)
Comment (jhelison) [reply libwallet/Cargo.toml]: We are keeping the authors just to Epic, such as is since we forked the repo

File: libwallet/Cargo.toml (hunk 12 above)
Comment (jhelison) [reply libwallet/Cargo.toml]: Done

File: impls/Cargo.toml (hunk 11 above)
Comment (jhelison) [reply impls/Cargo.toml]: Done

File: util/Cargo.toml (hunk 10 above)
Comment (jhelison) [reply util/Cargo.toml]: We are keeping the authors just to Epic, such as is since we forked the repo

File: controller/Cargo.toml (hunk 9 above)
Comment (jhelison) [reply controller/Cargo.toml]: We are keeping the authors just to Epic, such as is since we forked the repo

File: controller/Cargo.toml (hunk 8 above)
Comment (jhelison) [reply controller/Cargo.toml]: Done

File: config/Cargo.toml (hunk 7 above)
Comment (jhelison) [reply config/Cargo.toml]: Done

File: config/Cargo.toml (hunk 6 above)
Comment (jhelison) [reply config/Cargo.toml]: Done

File: config/Cargo.toml (hunk 5 above)
Comment (jhelison) [reply config/Cargo.toml]: We are keeping the authors just to Epic, such as is since we forked the repo

File: api/Cargo.toml (hunk 4 above)
Comment (jhelison) [reply api/Cargo.toml]: Done

File: api/Cargo.toml (hunk 3 above)
Comment (jhelison) [reply api/Cargo.toml]: We are keeping the authors just to Epic, such as is since we forked the repo

File: Cargo.toml (hunk 2 above)
Comment (jhelison) [reply Cargo.toml]: Done

File: Cargo.toml (hunk 1 above)
Comment (jhelison) [reply Cargo.toml]: We are keeping the authors just to Epic, such as is since we forked the repo

**Review by paultergust — APPROVED** [review]

**Review by paultergust — APPROVED** [review]

**Review by Jualns — COMMENTED** [review]
LGTM!

**Pull request merged**